hw-event-proxy/go.mod

```diff
@@ -4,7 +4,7 @@ go 1.17
 
 require (
 	github.com/json-iterator/go v1.1.12
-	github.com/redhat-cne/sdk-go v0.1.1-0.20220526200012-719244ead29f
+	github.com/redhat-cne/sdk-go v0.1.1-0.20220822193014-8b3ad5ad0a44
```

Comment:
we should try to make this point to latest every time it is build, that have we can reduce PR's. What do you think?

Reply:
we could use tags like v1.0.0